charts/fluent-bit/templates/_pod.tpl

```diff
@@ -9,6 +9,8 @@ priorityClassName: {{ .Values.priorityClassName }}
 serviceAccountName: {{ include "fluent-bit.serviceAccountName" . }}
 securityContext:
   {{- toYaml .Values.podSecurityContext | nindent 2 }}
+dnsConfig:
+  {{- toYaml .Values.podDnsConfig | nindent 2 }}
```

Comment:
```suggestion
{{- if .Values.dnsConfig }}
dnsConfig:
  {{- toYaml .Values.podDnsConfig | nindent 2 }}
{{- end }}
```

nit: lets wrap this in an `if` (or `with`) statement so we do not render `dnsConfig: {}`